src/views/Home.vue

```diff
@@ -87,9 +87,10 @@ export default {
     const favorites = computed(() => store.state.favoriteSpaces.favorites);
     const stateSpaces = computed(() => store.state.app.spaces);
 
-    const q = ref(route.query.q || '');
-
     const spaces = computed(() => {
+      const searchString = route.query.q || '';
```

Comment:
@ChaituVR why don't you use `const q = ref(route.query.q || '')` for `searchString` and `networkFilter` like it was before? Is there a reason for them to be inside the `space` computed property?

Reply:
@samuveth yes those values need to be inside the `space` computed value, to fix the issue where the snapshot logo is clicked and spaces with network/search filters don't clear, so it can take updated value from the route, 

previously when it was outside, computed `space` is not considering values changed in url

Reply:
> @bonustrack Yes, when you click on a network and then click on "snapshot" to go back to home, it gets stuck on that network.

i mean to fix issue